openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -813,7 +813,7 @@ The JWT MUST contain the following elements:
   * `typ`: REQUIRED. MUST be `openid4vci-proof+jwt`, which explicitly types the key proof JWT as recommended in Section 3.11 of [@!RFC8725].
   * `kid`: OPTIONAL. JOSE Header containing the key ID. If the Credential shall be bound to a DID, the `kid` refers to a DID URL which identifies a particular key in the DID Document that the Credential shall be bound to. It MUST NOT be present if `jwk` is present.
   * `jwk`: OPTIONAL. JOSE Header containing the key material the new Credential shall be bound to. It MUST NOT be present if `kid` is present.
-  * `x5c`: OPTIONAL. JOSE Header containing a certificate or certificate chain corresponding to the key used to sign the JWT. This element MAY be used to convey a key attestation. In such a case, the actual key certificate will contain attributes related to the key properties.
```

Comment:
why is `x5c` being deleted? `key_attestation` parameter should be added in addition to `x5c`, not instead of it

Reply:
Can you elaborate what the x5c field would be used for if we use the `key_attestation` field for attestations? Or do you see this as just an alternative place for key attestations?

Reply:
first of all, this is a `jwt` proof type section, so if a new proof type `attestation` is being defined, this section should stay unchanged. 

this is a place that does not define what is used with `key_attestation`, but what are the mechanisms that can be used in general, and right now it says, kid or jwk or x5c, so that should not change with this PR

Reply:
ok, i read the rest of the text and am confused/concerned. key attestation is being added to jwt proof type (so key attestation per proof), but a new key attestation proof type is also being introduced (key attestation per multiple proofs). i think we should not add key attestation to jwt proof type if ther eis a new key attestation.

Reply:
I disagree and we had a different agreement last time

Reply:
we left x5c in now with a stripped down version not referencing to key attestation